src/com/aca/Armine Arzumanyan/HomeWork_1/src/Gadget.java

```diff
@@ -0,0 +1,57 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Gadget implements Item {
+    public String name;
+    public int code;
+    public int price;
+    public int recDuration;
+    public boolean mp4;
+    public boolean mp3;
+    public boolean hd;
+    public boolean vga;
+    static List<Gadget> allItems = new ArrayList<>();
+    static List<Gadget> soledGadgets = new ArrayList<>();
+
+    public static void sailItem(Gadget item) {
```

Comment:
Does the gadget sale item? 

please look one more time **single-responsibility** principle in SOLID. You need additional model like a Manager and move these methods into that model